rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Layout:
+  Enabled: false
+
+Style/MultilineIfModifier: # clashes with rubyModifier
+  Enabled: false
+Style/SymbolArray: # clashes with rubyArrayLiteral
```

Comment:
For the default behavior, we don't need to disable this or the next rule. Prettier isn't forcing you into it, even if you disable `rubyArrayLiteral` you can still use array literals if you want to, it just won't transform them for you.